### PR TITLE
Change all scss imports to module style imports

### PR DESCRIPTION
**Pull request by peterhass**
There is a bug in `rollup-plugin-postcss` preventing this to work. https://github.com/egoist/rollup-plugin-postcss/issues/333

❌ ~Wait with merge until the bug is fixed~

✅  I published the fix under nebenan-rollup-plugin-postcss and added it to our npm organization. Will revert as soon as they accepted my PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "rollup-plugin-delete": "^2.0.0",
     "rollup-plugin-glob-import": "^0.4.5",
     "rollup-plugin-peer-deps-external": "^2.2.3",
-    "rollup-plugin-postcss": "^3.1.8",
```
Comment (peterhass) [inline package.json:72]: tbh I cannot find any reference to that version in their repo. Tried to build all our packages and it didn't cause any errors - should be good imo

Comment (suprMax) [reply package.json:72]: not sure what you mean, why not publish nebenan-rollup-plugin-postcss@1.0.0

Comment (peterhass) [reply package.json:72]: https://github.com/egoist/rollup-plugin-postcss/blob/master/package.json

no branch in their repo mentions version 3.1.8

I used 3.0.1 to keep their version scheme - would be weird to update later from nebenan-rollup-plugin-postcss 1.0.0 to rollup-plugin-postcss 3.1.x

Comment (suprMax) [reply package.json:72]: why? it's a different package. it's all good

Comment (peterhass) [reply package.json:72]: is it worth changing it? would need to unrelease + re-release

Comment (suprMax) [reply package.json:72]: i think it's better to avoid confision

**Review by suprMax — APPROVED** [review]

**Pull request merged**